ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -313,6 +315,16 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
         return result;
     }
 
+    public listUserLimits(): Promise<NamedRateStatus[]> {
```

Comment:
nit: `async` is easier to read then `.then` chaining.

Reply:
Done